src/adapter/src/coord/sequencer.rs

```diff
@@ -4028,7 +4031,10 @@ impl Coordinator {
             | Plan::Raise(_)
             | Plan::RotateKeys(_) => {
                 return Err(AdapterError::Unauthorized(
-                    "user 'mz_introspection' is unauthorized to perform this action".into(),
+                    rbac::UnauthorizedError::privilege(
+                        plan.name().to_string(),
+                        format!("You must not be the '{}' role", MZ_INTROSPECTION_ROLE.name),
```

Comment:
This error message feels a bit weird because it's requiring you aren't something instead of are something. Can the text be more descriptive, or just removed?

Reply:
I removed the text.